aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -70,10 +76,12 @@ Tap **F5** to run the app in debug mode or **Ctl-F5** in non-debug mode.
 
 ![IIS Express](start-mvc/_static/iis_express.png)
 
-The default template gives you working **Home, Contact, About, Register** and **Log in** links. The browser image above doesn't show these links. Depending on the size of your browser, you might need to click the navigation icon to show them.
+The default template gives you working **Home, About** and **Contact** links. The browser image above doesn't show these links. Depending on the size of your browser, you might need to click the navigation icon to show them.
 
 ![navigation icon in upper right](start-mvc/_static/2.png)
 
+Close the browser window, and if you were running in debug mode, tap **Shift-F5** to stop debugging.
```

Comment:
No to closing the browser. You can tell them "if debugging stop " once in the tutorial.